astrosat_users/tests/factories.py

```diff
@@ -128,8 +128,12 @@ class Meta:
 
     customer_type = CustomerType.MULTIPLE
 
-    name = factory.LazyAttributeSequence(lambda o, n: f"{slugify(o.title)}-{n}")
-    title = FactoryFaker("pretty_sentence", nb_words=2)
+    name = FactoryFaker("name")
+    official_name = FactoryFaker("name")
+    company_type = CompanyTypes.NON_PROFIT
+    registered_id = factory.LazyAttributeSequence(
+        lambda o, n: f"{slugify(o.name)}-{n}"
+    )
```

Comment:
I would have assumed more bits of this module would have changed.  There are some references to `generate` that could be replaced.  But, I guess that if you pinned **factoryboy** then this didn't cause an error.

Reply:
Actually I didn't pin factoryboy, leaving that as a separate task when adding the other dev-packages we discussed yesterday.  The only thing I did to `Pipefile` was to add a `scripts` block, like we have in the frontend, which allows us to wrap, potentially quite complex command, or mask a particular technology e.g. you can now run `pipenv run test` instead of `pipenv run pytest`, so we could change tech under the hood and still run the same command to test. You probably don't care about that, but I have got into the habit with using stuff like this in Java and Node.

Reply:
So what happened to the bug you were having w/ `locale` yesterday?

Reply:
That was because I had to delete the `Pipfile.lock` to be able to install the likes of `pytest-watch`, `yapf`, `pylint` and `ptvsd` dev-dependencies. Having removed them again and reverted the `Pipfile.lock`, I don't get any problems.